assets/src/edit-story/app/media/media3p/api/useMedia3pApi.js

```diff
@@ -20,6 +20,17 @@
 import { identity, useContextSelector } from '../../../../utils/context';
 import Context from './context';
 
+/** @typedef {import('./typedefs').Media3pApiContext} Media3pApiContext */
+
+/**
+ * Context value consumer to select a fragment of the context from
+ * {@link ./media3pApiProvider}.
+ *
+ * @param {function(Media3pApiContext):Object?} selector Returns a fragment of
+ * the media3p api context value that the caller is interested in. If no
+ * selector is provided then the entire context is selected.
+ * @return {Object} The selected context value fragment.
```

Comment:
This part is interesting. The 'selector' isn't actually used anywhere, and probably doesn't provide much value since the functions provided by Media3pProvider don't have any state that can trigger a re-render.

Optional. To get more value from the typedef, the selector can be removed (by simply removing the 'selector' parameter from here) and updating the return type from 'Object' to Media3pApiContext.

Reply:
Thats a good idea, I've removed the selector.